package.json

```diff
@@ -1,101 +1,101 @@
-    "http": "0.0.0",
-    "http-proxy-middleware": "0.21.0",
-    "js-string-escape": "1.0.1",
-    "lodash": "4.17.21",
-    "optional-require": "1.0.0",
-    "requirejs": "2.3.6",
-    "serve-static": "1.13.2",
-    "webpack": "4.44.1",
-    "which": "1.3.1"
-  },
-  "peerDependencies": {
-    "karma": ">= 1.7.0",
-    "karma-chrome-launcher": "2.1.1",
-    "karma-coverage": "1.1.1",
-    "karma-mocha": "1.3.0",
-    "karma-mocha-reporter": "2.2.3",
-    "karma-openui5": "0.2.2",
-    "karma-requirejs": "1.1.0",
-    "mocha": "*"
-  },
-  "devDependencies": {
-    "chai": "4.2.0",
-    "chai-as-promised": "7.1.1",
-    "chai-string": "1.5.0",
-    "eslint": "6.8.0",
-    "eslint-config-airbnb-base": "14.0.0",
-    "eslint-config-prettier": "6.9.0",
-    "eslint-plugin-import": "2.20.0",
-    "gitty": "3.6.0",
-    "jest": "23.6.0",
-    "jsonfile": "5.0.0",
-    "npm-run-all": "4.1.3",
-    "prettier": "1.14.3",
-    "semver": "5.6.0",
-    "shx": "0.3.2",
-    "typedoc": "0.16.8",
-    "typescript": "3.9.5",
-    "vuepress": "1.8.2"
-  },
-  "resolutions": {
-    "**/mixin-deep": "^1.3.2",
-    "**/set-value": "^2.0.1"
-  },
-  "scripts": {
-    "ci_full_flow": "npm-run-all format_validate type_check lint test docs website:build",
-    "release": "bash ./scripts/release.sh",
-    "lint": "eslint src/**/*.js test/**/*.js test_integration/**/*.js",
-    "test": "jest",
-    "test_integration": "npm-run-all init_example execute_integration_tests unlink_example",
-    "init_example": "node scripts/init_example.js",
-    "execute_integration_tests": "jest --maxWorkers 1 --config test_integration/jest.config.json",
-    "unlink_example": "cd ./example && npm unlink @sap-webide/webide-client-tools",
-    "docs": "shx rm -rf docs/web/html_docs && typedoc --tsconfig tsdocsconfig.json --out docs/web/html_docs --module commonjs --name WebideClientTools --excludeExternals --includeDeclarations",
-    "format": "prettier --write \"**/*.@(js|json|md)\"",
-    "format_validate": "prettier --list-different \"**/*.@(js|json|md)\"",
-    "type_check": "tsc",
-    "website:dev": "vuepress dev docs",
-    "website:build": "vuepress build docs"
-  },
-  "files": [
-    "src",
-    "docs",
-    "example",
-    "resources",
-    "README.md",
-    "api.d.ts"
-  ],
-  "prettier": {
-    "semi": false
-  },
-  "jest": {
-    "testEnvironment": "node",
-    "testRegex": "test/.*spec\\.js$",
-    "collectCoverage": true,
-    "coverageThreshold": {
-      "global": {
-        "branches": 100,
-        "functions": 100,
-        "lines": 100,
-        "statements": 100
-      }
-    },
-    "coveragePathIgnorePatterns": [
-      "<rootDir>/src/dev_server.js",
-      "<rootDir>/src/middleware.js",
-      "<rootDir>/src/di_backend.js"
-    ]
-  },
-  "typings": "api.d.ts"
-}
+{
+  "name": "@sap-webide/webide-client-tools",
+  "version": "3.1.3",
+  "license": "Apache-2.0",
+  "main": "src/api.js",
+  "dependencies": {
+    "@types/webpack": "5.28.0",
+    "acorn": "8.6.0",
```

Comment:
where there any breaking changes from acron 7 --> 8?

Reply:
you probably skipped it in the code https://github.com/SAP/webide-client-tools/blob/15e4fef4dc90cf44152c4e3c59d146c7d7588822/src/bundling.js#L749